src/concrete/ml/quantization/base_quantized_op.py

```diff
@@ -700,7 +700,7 @@ def _prepare_inputs_with_constants(
 
         return prepared_inputs
 
-    def calibrate(self, *inputs: numpy.ndarray) -> numpy.ndarray:
+    def calibrate(self, *inputs: Union[QuantizedArray, numpy.ndarray]) -> numpy.ndarray:
```

Comment:
Allow calibration with `QuantizedArray`. Thus the calibration can be performed analytically. 

this is similar to what Brevitas does with `QuantTensor` which determines output quant parameters based on the input ones, without calibration.